src/main.rs

```diff
@@ -32,7 +32,13 @@ mod version;
 fn main() {
     env_logger::init().unwrap_or_else(|e| panic!("{}", e));
 
-    let cfg = config::Config::new();
+    let cfg = match config::Config::new() {
+        Ok(c) => c,
+        Err(e) => {
+            println!("{}", e);
```

Comment:
I don't think I did. It was phrased suboptimally once but after that I was surprised you removed all the text.